dist/pom.xml

```diff
@@ -577,7 +577,6 @@ self.log("... OK")
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-install-plugin</artifactId>
-                <version>3.0.1</version>
```

Comment:
Any idea why this version was originally requested?  Curious if there was a historic reason to pick this specific version here and if we're going to expose a known bug by rolling back.

Reply:
Ouch, thanks for catching. Pretty sure I needed this for pomFile read from the jar.